app/assets/javascripts/max-attendances.js

```diff
@@ -0,0 +1,23 @@
+$(function() {
+    var fieldsCount,
+        maxFieldsCount = 2,
+        $addLink = $('a.add_nested_fields');
```

Comment:
JS / jQuery n00b question: the `$` only denotes that it's a jQuery DOM object, right? It's not some magic JS trick?